addons/website/static/src/js/content/menu.js

```diff
@@ -178,8 +178,10 @@ const BaseAnimatedHeader = animations.Animation.extend({
         }
 
         // Close opened menus
-        this.$dropdowns.removeClass('show');
-        this.$navbarCollapses.removeClass('show').attr('aria-expanded', false);
+        if (scroll) {
```

Comment:
I guess it's an acceptable solution... but you could still edit mega menus with the page scrolled. But if there really is no other way, it's ok, however: is not a comment useful to understand why this is necessary?

Reply:
I changed for not closing the mega menu on a scroll, I had not thought about it, thanks !